### PR TITLE
fix(test): add unique id for UI element, replace brittle xpath ref with new id

### PR DESCRIPTION
**Pull request by ldimaggi**
add unique id for UI element, replace brittle xpath ref with new id

### CODE REVIEW

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2314-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2314-fabric8-planner.badger.fabric8.io

File: runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js [hunk 1]
```diff
@@ -28,7 +28,8 @@ class WorkItemDetailPage {
   }
 
   get workItemDetailFullPageCloseButton () {
-    return element(by.xpath(".//*[contains (@class,'pficon-close pull-right margin-right-5 padding-15 close')]"));
+//    return element(by.xpath(".//*[contains (@class,'pficon-close pull-right margin-right-5 padding-15 close')]"));
```
Comment (jarifibrahim) [inline runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js]: @ldimaggi We can just delete this line, can't we?

Comment (ldimaggi) [reply runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js]: Yes!

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @ldimaggi for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by jarifibrahim** [comment]
Closing this PR since we're no longer using the old JS based tests.

**Pull request closed without merging**